### PR TITLE
Word count 2

### PR DESCRIPTION
**Pull request by amymhaddad**
Please only review the refactored version: refactor_word_count_2.py

### CODE REVIEW

File: python/word-count/refactor/refactor_word_count_2.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        underscores = has_special_chars(phrase)
+        if underscores:
+                return phrase
+        else:
+                exclude = set(string.punctuation)
+                exclude.add('\n')
+                exclude.remove(',')
+                exclude.remove("\'")
+        
+                updated_phrase = ''.join(character.lower().replace('\t', " ") for character in phrase if character not in exclude)
```
Comment (paulghaddad) [inline python/word-count/refactor/refactor_word_count_2.py]: Wow, impressive!

File: python/word-count/refactor/refactor_word_count_2.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        else:
+                return default(phrase)
+
+
+def standardize_phrase(phrase):
+        underscores = has_special_chars(phrase)
+        if underscores:
+                return phrase
+        else:
+                exclude = set(string.punctuation)
```
Comment (paulghaddad) [inline python/word-count/refactor/refactor_word_count_2.py]: Remember to indent one tab...something may be off in your editor.

File: python/word-count/refactor/refactor_word_count_2.py [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+def has_comma(phrase):
+        phrase = standardize_phrase(phrase)
+        return re.search(r',', phrase)
+
+
+def comma(phrase):
+        phrase = standardize_phrase(phrase)
+        has_comma(phrase)
+        return Counter(phrase.split(','))
```
Comment (paulghaddad) [inline python/word-count/refactor/refactor_word_count_2.py]: 👍

**Comment by amymhaddad** [comment]
@paulghaddad Yes, it's my editor. This happens often and I'm not sure why nor how to fix it.

**Pull request closed without merging**